data/risks/networks/10/strategylender.json

```diff
@@ -13,6 +13,8 @@
 			"genericlender"
 		],
 		"strategies": [],
-		"exclude": []
+		"exclude": [
```

Comment:
this one is needed only for mainnet at the moment

Reply:
alright, will leave the file in optimism